week5/days_in_a_year_test.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+
+
+class Test(unittest.TestCase):
+    
+    def test_isLeapYear(self):
+        self.assertEqual(isLeapYear(2000), True)
+        self.assertEqual(isLeapYear(2001), False)
+
+    def test_numberOfDaysInAYear(self):
+        self.assertEqual(numberOfDaysInAYear(2000), 366)
+        self.assertEqual(numberOfDaysInAYear(2001), 365)
+
+    def test_numberOfDaysInRange(self):
+        self.assertEqual(numberOfDaysInRange(2000, 2001), 366)
+
+if __name__ == "__main__":
+
+    from days_in_a_year import (
+        isLeapYear, numberOfDaysInAYear, numberOfDaysInRange
+    )
```

Comment:
in general, it is better to put the import at the top of the file so that we know what is imported is all at the same place (for readibility)